frontend/src/pages/profile/MemberDetailsSection.tsx

```diff
@@ -0,0 +1,26 @@
+import Detail from './Detail';
+
+const MemberDetailsSection = ({
+  numColumns,
+  title,
+  columns,
+}: {
+  numColumns: number;
+  title: string;
+  columns: { title: string; content: string }[];
+}) => {
+  return (
+    <div className="py-5">
+      <h4 className="font-bold text-info">{title}</h4>
+      <div className={`grid grid-cols-${numColumns} pt-4`}>
+        {columns.map((column, i) => (
+          <div key={i}>
```

Comment:
warning for using index as key . Not breaking, but maybe we can add the value + i.toString or something? I really don't know what the best practice is here.